src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -1,29 +1,27 @@
 package org.jenkinsci.plugins.gitclient;
 
-
 import com.cloudbees.jenkins.plugins.sshcredentials.SSHUserPrivateKey;
 import com.cloudbees.plugins.credentials.common.StandardCredentials;
-import com.cloudbees.plugins.credentials.common.UsernameCredentials;
 import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
-
+import com.google.common.collect.Lists;
```

Comment:
The [contributing guidelines](https://github.com/jenkinsci/git-client-plugin/blob/master/CONTRIBUTING.md) for the plugin ask that contributors not perform wholesale reformatting of files without consulting the maintainers.  Moving the location of an import seems like it leans toward wholesale reformatting.  Couldn't this be put back in the location where it was prior to your change, so that the comparison between versions is smaller?